app-editors/vim-core/vim-core-9.0.1341.ebuild

```diff
@@ -0,0 +1,233 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Please bump with app-editors/vim and app-editors/gvim
+
+VIM_VERSION="9.0"
+VIM_PATCHES_VERSION="9.0.1000"
+inherit bash-completion-r1 desktop flag-o-matic prefix toolchain-funcs vim-doc xdg-utils
+
+if [[ ${PV} == 9999* ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/vim/vim.git"
+	EGIT_CHECKOUT_DIR=${WORKDIR}/vim-${PV}
+else
+	SRC_URI="https://github.com/vim/vim/archive/v${PV}.tar.gz -> vim-${PV}.tar.gz
+		https://gitweb.gentoo.org/proj/vim-patches.git/snapshot/vim-patches-vim-${VIM_PATCHES_VERSION}-patches.tar.bz2"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~x64-cygwin ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+fi
+S="${WORKDIR}/vim-${PV}"
+
+DESCRIPTION="vim and gvim shared files"
+HOMEPAGE="https://www.vim.org https://github.com/vim/vim"
+
+LICENSE="vim"
+SLOT="0"
+IUSE="nls acl minimal"
+
+# ncurses is only needed by ./configure, so no subslot operator required
+DEPEND=">=sys-libs/ncurses-5.2-r2:0"
+BDEPEND="sys-devel/autoconf"
+
+if [[ ${PV} != 9999* ]]; then
+	# Gentoo patches to fix runtime issues, cross-compile errors, etc
+	PATCHES=(
+		"${WORKDIR}/vim-patches-vim-${VIM_PATCHES_VERSION}-patches"
+	)
+fi
+
+pkg_setup() {
+	# people with broken alphabets run into trouble. bug #82186.
+	unset LANG LC_ALL
+	export LC_COLLATE="C"
+}
+
+src_prepare() {
+	default
+
+	# Fixup a script to use awk instead of nawk
+	sed -i \
+		-e '1s|.*|#!'"${EPREFIX}"'/usr/bin/awk -f|' \
+		"${S}"/runtime/tools/mve.awk || die "sed failed"
+
+	# See bug #77841. We remove this file after the tarball extraction.
+	rm -v "${S}"/runtime/tools/vimspell.sh || die "rm failed"
+
+	# Read vimrc and gvimrc from /etc/vim
+	echo '#define SYS_VIMRC_FILE "'${EPREFIX}'/etc/vim/vimrc"' >> "${S}"/src/feature.h || die
+	echo '#define SYS_GVIMRC_FILE "'${EPREFIX}'/etc/vim/gvimrc"' >> "${S}"/src/feature.h || die
+
+	# Use exuberant ctags which installs as /usr/bin/exuberant-ctags.
+	# Hopefully this pattern won't break for a while at least.
+	# This fixes bug #29398 (27 Sep 2003 agriffis)
+	sed -i 's/\<ctags\("\| [-*.]\)/exuberant-&/g' \
+		"${S}"/runtime/doc/syntax.txt \
+		"${S}"/runtime/doc/tagsrch.txt \
+		"${S}"/runtime/doc/usr_29.txt \
+		"${S}"/runtime/menu.vim \
+		"${S}"/src/configure.ac || die 'sed failed'
+
+	# Don't be fooled by /usr/include/libc.h.  When found, vim thinks
+	# this is NeXT, but it's actually just a file in dev-libs/9libs
+	# This fixes bug #43885 (20 Mar 2004 agriffis)
+	sed -i 's/ libc\.h / /' "${S}"/src/configure.ac || die 'sed failed'
```

Comment:
I don't know about this one either, has this sed been carried for 20 years?
```
 * SED: the following did not cause any changes
 *     sed -i 's/ libc\.h / /' "${S}"/src/configure.ac || die 'sed failed';
 * no-op: -e s/ libc\.h / /
```

Same for vim and gvim.

Reply:
It looks like the `libc.h` mentioned in the bug has been moved to `9libs/libc.h`. It still gives me "checking for libc.h... no" after installing `dev-libs/9libs` and doing a round of `ebuild ... clean configure`. So this doesn't appear to be relevant anymore.

As a side note: the check still exists in `src/configure.ac`, it's just moved and has a hard tab in front of it now instead of a space which is why nothing was changed by portage.